guides/source/debugging_rails_applications.md

```diff
@@ -436,13 +436,12 @@ It's also possible to use these options together: `backtrace [num] /pattern/`.
 
 #### The outline command
 
-This command is similar to `pry` and `irb`'s `ls` command. It will show you what's accessible from you current scope, including:
+This command is similar to `pry` and `irb`'s `ls` command. It will show you what's accessible from the current scope which among others, include:
```

Comment:
Hmm, "including:" still reads more naturally to me.

```suggestion
This command is similar to `pry` and `irb`'s `ls` command. It will show you what's accessible from the current scope, including:
```